docs/LICENSE.md

```diff
@@ -0,0 +1,7 @@
+Copyright (C) 2015 by Haris Amin
```

Comment:
2017 – Copyright attribution should perhaps be "Swiftdocker maintainers"? @hamin ?